apps/asap-server/test/controllers/events.test.ts

```diff
@@ -1,920 +1,643 @@
-    test('Should return permanentlyUnavailable details if details are permanentlyUnavailable on the CMS', async () => {
-      const pUnavailableEventRes = eventBase;
-      pUnavailableEventRes.flatData!.endDate = '2009-12-24T16:20:14Z';
-      pUnavailableEventRes.flatData!.notesPermanentlyUnavailable = true;
-      pUnavailableEventRes.flatData!.videoRecordingPermanentlyUnavailable =
-        true;
-      pUnavailableEventRes.flatData!.presentationPermanentlyUnavailable = true;
-      pUnavailableEventRes.flatData!.meetingMaterialsPermanentlyUnavailable =
-        true;
+      test('Should return permanentlyUnavailable details if details are permanentlyUnavailable on the CMS', async () => {
+        const eventsGraphqlResponse = getSquidexEventsGraphqlResponse();
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.notesPermanentlyUnavailable =
+          true;
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.videoRecordingPermanentlyUnavailable =
+          true;
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.presentationPermanentlyUnavailable =
+          true;
+        eventsGraphqlResponse.queryEventsContentsWithTotal!.items![0]!.flatData.meetingMaterialsPermanentlyUnavailable =
+          true;
+
+        squidexGraphqlClientMock.request.mockResolvedValueOnce(
+          eventsGraphqlResponse,
+        );
 
-      nock(config.baseUrl)
-        .post(`/api/content/${config.appName}/graphql`, {
-          query: print(FETCH_EVENT),
-          variables: {
-            id: eventId,
-          },
-        })
-        .reply(200, {
-          data: { findEventsContent: pUnavailableEventRes },
+        const result = await eventsController.fetch({
+          before: 'before',
         });
-
-      const result = await events.fetchById(eventId);
-      expect(result).toEqual({
-        ...eventResponse,
-        notes: null,
-        videoRecording: null,
-        presentation: null,
-        meetingMaterials: null,
+        expect(result.items[0]?.notes).toBeNull();
+        expect(result.items[0]?.videoRecording).toBeNull();
+        expect(result.items[0]?.meetingMaterials).toBeNull();
       });
-    });
 
-    test('Should return permanentlyUnavailable details if enough time has passed and details are empty', async () => {
-      const emptyEvent = eventBase;
-      emptyEvent.flatData!.notes = null;
-      emptyEvent.flatData!.videoRecording = null;
-      emptyEvent.flatData!.presentation = null;
-      emptyEvent.flatData!.meetingMaterials = null;
-      emptyEvent.flatData!.notesPermanentlyUnavailable = null;
-      emptyEvent.flatData!.videoRecordingPermanentlyUnavailable = null;
-      emptyEvent.flatData!.presentationPermanentlyUnavailable = null;
-      emptyEvent.flatData!.meetingMaterialsPermanentlyUnavailable = null;
+      test('Should return permanentlyUnavailable details if enough time has passed and details are empty', async () => {
```

Comment:
I'm not sure how the 'enough time has passed' criteria is actually met here, do you know what it is referring to? maybe we should be more explicit with the fixtures somehow or i dont know

Reply:
I think these tests are testing this code
https://github.com/yldio/asap-hub/blob/master/apps/asap-server/src/entities/event.ts#L20-L24

The tests didn't look right, so thank you! I updated them. See the last commit